### PR TITLE
add --labels

### PR DESCRIPTION
**Pull request by danielnorberg**

### CODE REVIEW

File: spydra/src/main/java/com/spotify/spydra/submitter/runner/SubmissionCliParser.java [hunk 1]
```diff
@@ -88,6 +89,14 @@ public SpydraArgument parse(String[] args) throws IOException {
           sanitizeJobId(cmdLine.getOptionValue(CliConsts.JOBNAME_OPTION_NAME)));
     }
 
+    if (cmdLine.hasOption(CliConsts.LABELS_OPTION_NAME)) {
+      String labels = StringUtils.join(cmdLine.getOptionValues(CliConsts.LABELS_OPTION_NAME), ",");
+      spydraArgument.getSubmit().getOptions().compute(SpydraArgument.OPTION_LABELS,
+          (key, existingLabels) -> (existingLabels == null) ? labels : existingLabels + '.' + labels);
+      spydraArgument.getCluster().getOptions().compute(SpydraArgument.OPTION_LABELS,
+          (key, existingLabels) -> (existingLabels == null) ? labels : existingLabels + '.' + labels);
```
Comment (Xeago) [inline spydra/src/main/java/com/spotify/spydra/submitter/runner/SubmissionCliParser.java]: Change of separator to `.` compared to few lines above. Could you clarify the intent?

File: spydra/src/main/java/com/spotify/spydra/submitter/runner/SubmissionCliParser.java [hunk 2]
```diff
@@ -17,22 +17,20 @@
...
 import java.io.FileInputStream;
 import java.io.IOException;
 import java.util.LinkedList;
 import java.util.List;
 import java.util.UUID;
-
-import static java.lang.Integer.min;
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.DefaultParser;
+import org.apache.commons.cli.HelpFormatter;
```
Comment (Xeago) [inline spydra/src/main/java/com/spotify/spydra/submitter/runner/SubmissionCliParser.java]: Please auto-format in a separate commit. I believe the rest of the code spaces library imports from jdk imports.

File: spydra/src/test/java/com/spotify/spydra/submitter/runner/SubmissionCliParserTest.java [hunk 3]
```diff
@@ -59,4 +57,13 @@ public void testJobIdMaxLength() throws IOException {
     assertEquals(88, fixed_id.length());
   }
 
+
+  @Test
+  public void testLabels() throws IOException {
+    SubmissionCliParser parser = new SubmissionCliParser();
+    String[] args = {"--labels=k1=v1", "--labels=k2=v2,k3=v3"};
+    SpydraArgument argument = parser.parse(args);
```
Comment (Xeago) [inline spydra/src/test/java/com/spotify/spydra/submitter/runner/SubmissionCliParserTest.java]: Code in the call to compute is never tested and behaves odd in my opinion. Could you ensure converse of that?

File: spydra/src/main/java/com/spotify/spydra/submitter/runner/SubmissionCliParser.java (hunk 1 above)
Comment (danielnorberg) [reply spydra/src/main/java/com/spotify/spydra/submitter/runner/SubmissionCliParser.java]: good catch =)

File: spydra/src/main/java/com/spotify/spydra/submitter/runner/SubmissionCliParser.java (hunk 2 above)
Comment (danielnorberg) [reply spydra/src/main/java/com/spotify/spydra/submitter/runner/SubmissionCliParser.java]: Sorry about that, intellij likes to automatically optimize imports. It is configured with the latest google style though.

File: spydra/src/test/java/com/spotify/spydra/submitter/runner/SubmissionCliParserTest.java (hunk 3 above)
Comment (danielnorberg) [reply spydra/src/test/java/com/spotify/spydra/submitter/runner/SubmissionCliParserTest.java]: Adding tests.

**Comment by danielnorberg** [comment]
Rebased on top of #45

File: spydra/src/main/java/com/spotify/spydra/submitter/runner/SubmissionCliParser.java (hunk 2 above)
Comment (danielnorberg) [reply spydra/src/main/java/com/spotify/spydra/submitter/runner/SubmissionCliParser.java]: moved the import order change to a separate PR #45 and rebased this PR on that.

**Comment by ulzha** [comment]
Reflect in documentation as well?

**Comment by danielnorberg** [comment]
Closing in favor of #46.

**Pull request closed without merging**